exporter/kafkaexporter/config_test.go

```diff
@@ -42,6 +44,20 @@ func TestLoadConfig(t *testing.T) {
 			NameVal: typeStr,
 			TypeVal: typeStr,
 		},
+		TimeoutSettings: exporterhelper.TimeoutSettings{
+			Timeout: 10 * time.Second,
+		},
+		RetrySettings: exporterhelper.RetrySettings{
+			Enabled:         true,
+			InitialInterval: 10 * time.Second,
+			MaxInterval:     1 * time.Minute,
+			MaxElapsedTime:  10 * time.Minute,
+		},
+		QueueSettings: exporterhelper.QueueSettings{
```

Comment:
Are the default values based on something or just common sense (also for the retry) ?

Reply:
Just common sense. there is a documentation on how to configure the sending queue:
https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/exporterhelper/queued_retry.go#L43